.github/workflows/cmake-format-check.yml

```diff
@@ -13,6 +13,8 @@ jobs:
     - name: cmake-format lint action
       uses: puneetmatharu/cmake-format-lint-action@efbb497b2a8badd2c9dc638faaf8ef4a9aa71bc8 # v1.0.4
       with:
+        build-args: |
+            PROCESS_FILES: '*.cmake|*.cmake.in|CMakeLists.txt'
```

Comment:
I don't understand how build-args works.  Would you add a reference to code or documentation?

Reply:
I switched to `env` which should be more expressive and the more general approach
https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#defining-environment-variables-for-a-single-workflow